bigbluebutton-html5/imports/ui/components/debug-window/component.jsx

```diff
@@ -0,0 +1,185 @@
+import React, { PureComponent } from 'react';
+import cx from 'classnames';
+import Draggable from 'react-draggable';
+import styles from './styles.scss';
+import Modal from '/imports/ui/components/modal/simple/component';
+import { withModalMounter } from '/imports/ui/components/modal/service';
+import Toggle from '/imports/ui/components/switch/component';
+import Button from '/imports/ui/components/button/component';
+import Storage from '/imports/ui/services/storage/session';
+import { withLayoutConsumer } from '/imports/ui/components/layout/context';
+
+const SHOW_DEBUG_WINDOW = Meteor.settings.public.app.showDebugWindow;
+const SHOW_DEBUG_WINDOW_SHORTCUT = Meteor.settings.public.app.showDebugWindowShortcut;
+
+class DebugWindow extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dragging: false,
+      position: {
+        x: 0,
+        y: 0,
+      },
+    };
+
+    this.handleResize = this.handleResize.bind(this);
+    this.handleOnDrag = this.handleOnDrag.bind(this);
+    this.handleDragStop = this.handleDragStop.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this.handleResize, false);
+    document.addEventListener('keyup', (event) => {
+      const key = event.key.toUpperCase();
+      if (SHOW_DEBUG_WINDOW_SHORTCUT && event.ctrlKey && event.altKey && key === 'K') {
```

Comment:
Might be a good idea to have this shortcut follow some of the other shortcuts, using the `accesskey` attribute for consistency. It should also be visible in the keyboard shortcut modal.